packages/compiler/lib/lib.cadl

```diff
@@ -88,6 +88,9 @@ model boolean {}
 @intrinsic("null")
 model null {}
 
+@intrinsic("uri")
```

Comment:
did we decide to have this an intrinsic type and not a string with the uri format? For type relation should this be assignable to a string?